masterlist.yaml

```diff
@@ -10355,3 +10355,114 @@ plugins:
         condition: 'many("CaN.es(m|p)")'
   - name: 'CaN.esp'
     msg: [ *obsolete ]
+
+  - name: 'Pip-Boy Flashlight.esp'
+    url: [ 'https://www.nexusmods.com/fallout4/mods/10840' ]
+    group: *lightingGroup
+    after:
+      # Breaks pip-boy light entirely if loaded before ELFX
+      - 'EnhancedLightsandFX.esp'
+    msg:
+      # Recommended by mod author
+      - <<: *patch3rdParty
+        subs:
+          - 'Enclave X-02 Power Armor'
+          - '[X-02 PipBoy Flashlight Patch](https://www.nexusmods.com/fallout4/mods/16232)'
+        condition: 'active("EnclaveX02.esp") and not active("X-02 PipboyLight Patch.esp")'
+      - <<: *obsolete
+        condition: 'version("Pip-Boy Flashlight.esp", "5.0.2", <) and not version("Pip-Boy Flashlight.esp", "v6b3.02", ==)'
```

Comment:
These conditions won't have any effect, as there's no version info listed in the plugin header.

It would need to be listed in the description like below.

![image](https://user-images.githubusercontent.com/1944639/64484486-96061e00-d20a-11e9-8c6b-3edb65d6e3c5.png)

Reply:
Best to just remove the obsolete message, as the alternative of using checksums would be too awkward due to the number of plugin options.